roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -1,22 +0,0 @@
----
```

Comment:
Since task "creating record" is now pinned to Route53 zones we will have task & code duplication for Dyn. Dyn is an absolute req to remove the Ops blocker. Again not future proof.

Reply:
There is no dyn requirement.  I'm not sure you are following the flow of how the dns records are created.